src/main/java/io/xpire/commons/util/JsonUtil.java

```diff
@@ -100,6 +100,7 @@
         return objectMapper.readValue(json, instanceClass);
     }
 
+
```

Comment:
Extra line here haha.

Reply:
When I didn't leave the extra line here I got an error from Travis :thinking: